tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -202,4 +202,36 @@ Wait for release amendment process status to be Complete
 Go to permalink page & check for error element to be present
     [Tags]  HappyPath
     user goes to url  ${PERMA_LOCATION_URL}
-    user waits until page contains  WARNING - The data used in this permalink may be out-of-date.
\ No newline at end of file
+    user waits until page contains  WARNING - The data used in this permalink may be out-of-date.
+
+# EES-1303
```

Comment:
Same as above, I'd use `[Documentation]  EES-1303` on the test cases specifically related to that ticket - but we haven't done this elsewhere, so don't feel you must.